Blog/include/loginFunctions.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+// include('Blog/config/config.php');
+// include('Blog/include/db_query.php');
+
+// $userName = isset($_REQUEST['userName']);
+// $passWord = isset($_REQUEST['passWord']);
+
+
+function findUsers(){
+	$result = dbQuery("
+		SELECT *
+		FROM users
+	")->fetch();
+
+	return $result;
+}
+
+function findUser($userName){
+	var_dump($userName);
+	$result = dbQuery("
+		SELECT *
+		FROM users
+		WHERE userName = :userName
+	",
+	array("userName" => $userName)
+)->fetch();
+
+
+// var_dump($result);
+	return $result;
+}
+
+// function getDistinctUser($userName){
+// 	$result = dbQuery("
+// 		SELECT *
+// 		FROM users
+// 		WHERE userName = :userName
+// 	",
+// 	 array('userName' => $userName)
+//  )
+// 	->fetch();
+//
+// }
+
+function verifyUser($userName, $passWord){
+	$findUser = findUser($userName);
+	$userNme = $findUser['userName'];
+	$passWrd = $findUser['passWord'];
+	$usrID = $findUser['userID'];
+	// dbQuery("
+	// 	SELECT *
+	// 	FROM users
+	// 	WHERE userName = :userName
+	// 	AND passWord = :passWord
+	// ",
+	//  array('userName' => $userName,
+ // 	'passWord' => $passWord,)
+ // );
+
+	if ($userName == $userNme && $passWord == $passWrd){
+		$_SESSION['userID'] = $usrID;
+		header('Location: logInConfirmation.php');
+		exit();
+	}
+	else{
+		echo "Login Failed";
```

Comment:
This works, but a slightly cleaner way to do this is to have it set the session and return true, or just return false. Then you call it outside of the `if(sizeof($Errors))` part of the form submission, so if there's an error, you treat it like any other form validation error.